### PR TITLE
Style changes for the desktop's icon grid

### PR DESCRIPTION
**Pull request by mariospr**
This PR makes some changes to make the desktop look more EOS-ish, with the grid of icons properly spaced and centered and the icons being of the right size.

https://phabricator.endlessm.com/T17658

### CODE REVIEW

File: js/ui/appDisplay.js [hunk 1]
```diff
@@ -457,8 +457,8 @@ const AllView = new Lang.Class({
         panAction.connect('gesture-cancel', Lang.bind(this, this._onPanEnd));
         panAction.connect('gesture-end', Lang.bind(this, this._onPanEnd));
         this._panAction = panAction;
-        this._scrollView.add_action(panAction);
         this._panning = false;
+
```
Comment (joaquimrocha) [inline js/ui/appDisplay.js]: This newline does not belong to this patch..

Comment (mariospr) [inline js/ui/appDisplay.js]: I added it on purpose to separate these two blocks, as one is about panning and the other about folders. I'd like to keep it
